tests/test-class-module.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
 class WP_Test_Edit_Flow_Class_Module extends WP_UnitTestCase {
+
 	protected static $admin_user_id;
+	protected static $EditFlowModule;
 
 	public static function wpSetUpBeforeClass( $factory ) {
 		self::$admin_user_id = $factory->user->create( array( 'role' => 'administrator' ) );
+		self::$EditFlowModule = new EF_Module();
```

Comment:
Is `self::$EditFlowModule = null` necessary in the tearDown? (Just curious, don't know much about PHP GC)